js/index.js

```diff
@@ -1,7 +1,58 @@
 // Iteration 1: Names and Input
 
+let hacker1 = 'Jonathan';
+console.log("Driver's name is:" + hacker1);
+
+let hacker2 = 'Ozhan';
+console.log('Navigators name is:' + hacker2);
 
```

Comment:
nice job guys!
First comment:
I would recommend to keep consistency in the quoting for the strings which means, if you use this (" ") quotes use it for all the strings, or the same with the other type of quotes.
Also I would reccomend to try the 
```
`${variable}`
``` 
type of quoting. So in this case it would be 
```
`Drivers name is: ${hacker1}`
```
as an example